docs/video-miners/how-to-guides/l2-migration.md

```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
     
-    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address.
-    
-    <img src="/docs-assets/video-miners/how-to-guides/stake-info.png" alt="stake info" width="300" />
-
-
+    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amounts will not be accurate until after you sign the transaction and we autodetect your orchestrator account from the signature.
```

Comment:
The correct stake amounts will appear after you enter your public ethereum address. Also should this note can probably be moved underneath the "sign using cli" option?
![image](https://user-images.githubusercontent.com/555740/151040618-21653122-3eba-441a-b0e7-260e58ba788a.png)